src/main/resources/templates/db_scripts/schema.json

```diff
@@ -167,6 +172,18 @@
           "removeAccents": true
         }
       ],
+      "fullTextIndex": [
+      	{
+          "fieldName": "identifiers",
+          "arraySubfield" : "value",
+          "arrayModifiers": ["identifierTypeId"]
+        },
```

Comment:
As I understand it, the `identifiers` array is part of the `item` object for a request. This index appears to refer to a top level property. Have I missed something?

Reply:
It is, yes.  I was unsure if I needed to put this in some kind of hierarchical notation, or if the schema would simply locate the field anywhere in the hierarchy based on the provided name.  When I tried it this way, it seemed to work, so I assumed the latter.

Reply:
> It is, yes. I was unsure if I needed to put this in some kind of hierarchical notation, or if the schema would simply locate the field anywhere in the hierarchy based on the provided name

I think there is some hierarchical notation needed. I don't think the schema tries to match this against any property.

> When I tried it this way, it seemed to work, so I assumed the latter.

How did you know it worked?

Reply:
Since this code looks like it's connected to generating the index, if it weren't working, I would expect to see: 1) an error message in the test output indicating the field could not be found and 2) the test would fail because the index isn't properly generated, and the identifier field is therefore not searchable.  I'm assuming a hierarchical notation would look like "item.identifiers"?  If it's wrong, and the isbn index is still being generated (as it seems to be), then is that code necessary at all?  I actually tried removing that chunk entirely, and the search test seems to work without it.

Reply:
> Since this code looks like it's connected to generating the index, if it weren't working, I would expect to see: 1) an error message in the test output indicating the field could not be found and 2) the test would fail because the index isn't properly generated, and the identifier field is therefore not searchable. 

As I understand it, the CQL index that is being introduced here is the `isbn` index (created below) rather than an index on the identifiers in general. 

I imagine if this index was removed, the tests would still work as before as they are exercising that index rather than this one.

> I'm assuming a hierarchical notation would look like "item.identifiers"? If it's wrong, and the isbn index is still being generated (as it seems to be), then is that code necessary at all? I actually tried removing that chunk entirely, and the search test seems to work without it.

Ah, it seems you've come to the same conclusion as me above.